spec/unit/container_spec.rb

```diff
@@ -134,22 +131,22 @@ def run(input)
       let(:other_interceptor) { TestInterceptor.new("second") }
 
       before :each do
-        sut.register(:command) { command }.intercept(:run).with(interceptor).and(other_interceptor)
-        sut.register(:single_command) { command }.intercept(:run).with(interceptor)
-        sut.resolve(:command).run("hi")
+        subject.register(:command) { command }.intercept(:run).with(interceptor).and(other_interceptor)
+        subject.register(:single_command) { command }.intercept(:run).with(interceptor)
+        subject.resolve(:command).run("hi")
       end
 
-      it "should allow the first interceptor to intercept calls to the target" do
-        interceptor.called.should be_true
+      it "allows the first interceptor to intercept calls to the target" do
+        expect(interceptor.called).to be_truthy
       end
 
-      it "should allow the second interceptor to intercept calls to the target" do
-        other_interceptor.called.should be_true
+      it "allows the second interceptor to intercept calls to the target" do
+        expect(other_interceptor.called).to be_truthy
       end
 
-      it "should forward the args to the command" do
-        command.called.should be_true
-        command.received.should == ['hi']
+      it "forwards the args to the command" do
+        expect(command.called).to be_truthy
+        expect(command.received).to match_array(['hi'])
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.